packages/gatsby/src/redux/reducers/queries.ts

```diff
@@ -225,6 +225,11 @@ export function queriesReducer(
       return state
     }
     case `MERGE_WORKER_QUERY_STATE`: {
+      // This action may be dispatched in cases where queries might not be included in the merge data
+      if (!action.payload.queryStateChunk) {
```

Comment:
What made this start happening?

Reply:
I didn't run into this happening exactly, but because `queryStateChunk` is added conditionally in the worker pool manager code, I wanted to make sure we didn't break things if it wasn't present for some reason

https://github.com/gatsbyjs/gatsby/pull/35427/files#diff-39c5caa1451bc42018f6769f9a1d9eb8877a213fa340c7bf23404b0f3cdd606bR130

Reply:
I added a check for this in the `telemetry` reducer as well where we handle the same action: https://github.com/gatsbyjs/gatsby/pull/35427/files#diff-42f89f373af246dfb622f3f0824095d5840a7ce40f39b9f96b224aae8470899eR24-R27

Reply:
Ah got it, thanks for the explanation